torch/nn/quantized/modules/batchnorm.py

```diff
@@ -1,28 +1,24 @@
 import torch
 import torch.nn.quantized.functional
-import torch.nn.intrinsic as nni
 
-class BatchNorm2d(torch.nn.BatchNorm2d):
-    r"""This is the quantized version of :class:`~torch.nn.BatchNorm2d`.
-    """
 
+class _BatchNormBase(torch.nn.modules.batchnorm._BatchNorm):
```

Comment:
optional: the conv classes use `_ConvNd` naming, we could also use `_BatchNormNd` here.  Or, use `_BatchNorm` to be consistent with the parent class.  Would be great to not introduce one additional naming scheme.